Dockerfile

```diff
@@ -615,7 +615,7 @@ RUN \
   --enable-libdavs2 \
   --enable-libxavs2 \
   --enable-libmodplug \
-  --enable-libuavs3d \
+#  --enable-libuavs3d \
```

Comment:
try this maybe...

https://github.com/uavs3/uavs3d/commit/e58e199ccc50163c20df114db7e3950295c3e2ed.patch